packages/shared/src/hooks/useGenerateUsername.ts

```diff
@@ -13,9 +13,10 @@ export const useGenerateUsername = (
   name: string | undefined,
 ): UseGenerateUsername => {
   const [username, setUsername] = useState('');
+  const usernameRef = useRef(false);
   const { requestMethod } = useRequestProtocol();
-  const usernameQueryKey = ['generateUsername', name];
-  const { data, isLoading } = useQuery<{
+  const usernameQueryKey = ['generateUsername'];
```

Comment:
If this hook was re-used in another part of our webapp it would then have the same value as the first call even though the actual hook would be called with the new `name`, which would be wrong.

Reply:
@capJavert Let me know what you think of the change

Reply:
I think it has to include it because we built this hook as a feature hook that should be re-usable across the webapp. Someone could use it down the line and encounter bugs without even knowing.